src/Workspaces/CSharp/Portable/Extensions/SyntaxTriviaExtensions.cs

```diff
@@ -50,6 +50,11 @@ public static bool IsMultiLineComment(this SyntaxTrivia trivia)
             return trivia.Kind() == SyntaxKind.MultiLineCommentTrivia;
         }
 
+        public static bool IsShebangComment(this SyntaxTrivia trivia)
+        {
+            return trivia.Kind() == SyntaxKind.ShebangCommentTrivia;
```

Comment:
I think it's probably up to API czars. For XML doc comments, we have things like LessThanExclamationMinusMinus. However, this isn't a new token, so i't probably OK. @AnthonyDGreen?